packages/frontend/src/components/common/FieldSelect/index.tsx

```diff
@@ -203,6 +226,7 @@ const FieldSelect = <T extends Item = Item>({
 
     return (
         <Select
+            ref={inputRef}
```

Comment:
if I remember correctly, there should be `autofocus` prop on the component. can you try that out?

Reply:
![CleanShot 2024-11-25 at 18 19 35@2x](https://github.com/user-attachments/assets/0a9e1f2a-75d7-40ae-8227-da0b2c3fbb36)

It seems to have the prop but it doesn't do anything... am I missing something?

Reply:
hmm I found and tested this solution: https://stackoverflow.com/questions/69456631/how-do-i-set-autofocus-to-an-input-thats-inside-a-select-dropdown

but it re-focuses on every render which is not going to work

Reply:
gotcha